src/Orleans/Streams/Predicates/FilterPredicateWrapperData.cs

```diff
@@ -50,6 +50,9 @@ internal class FilterPredicateWrapperData : IStreamFilterPredicateWrapper
         [NonSerialized]
         private StreamFilterPredicate predicateFunc;
 
+        // constructor used by serializator
+        private FilterPredicateWrapperData() { }
```

Comment:
Are all consumers going to need to add parameterless constructors? Are we ok with that?